### PR TITLE
Improve authorization in README

### PR DESCRIPTION
**Pull request by ryanseys**
I added some global authorization examples to the README and tried to help out users that have a tendency to jump around the README and not read it start-to-finish (basically every user ever).

Also, is it authentication and authorization? Just authorization? I can never understand the difference. [This](https://cloud.google.com/compute/docs/authentication#using) says Authenticating but its under the section of Authorization so I'm really confused.

Feedback very much welcome with respect to wording, overall tone, clarity, amount of babying, etc.

This aims to address the authorization section of https://github.com/GoogleCloudPlatform/gcloud-node/issues/374

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +62,21 @@ If you are not running this client on Google Compute Engine, you need a Google D
...
 
+``` js
+// Authorizing on a global basis.
+var projectId = process.env.GCLOUD_PROJECT_ID; // E.g. 'grape-spaceship-123'
+var keyFile = process.env.GCLOUD_KEYFILE; // E.g. /path/to/keyfile.json
+
+var gcloud = require('gcloud')({
+  projectId: projectId,
+  credentials: require(keyFile)
+});
```
Comment (stephenplusplus) [inline README.md]: I would rather highlight using `keyFIlename` here, since we will do an async fetch and parse of the file.

**Comment by stephenplusplus** [comment]
It's perfect!

File: README.md (hunk 1 above)
Comment (ryanseys) [inline README.md]: Certainly!

**Comment by ryanseys** [comment]
PTAL

File: README.md [hunk 2]
```diff
@@ -50,6 +62,20 @@ If you are not running this client on Google Compute Engine, you need a Google D
...
+
+var gcloud = require('gcloud')({
+  projectId: projectId,
+  keyFilename: '/path/to/keyfile.json'
+});
+
+// ...you're good to go! See the next section to get started using the APIs.
+```
+
+You can also set auth on a per-API-instance basis. The examples below show you
```
Comment (stephenplusplus) [inline README.md]:
```diff
-show you
+show you.
```

or maybe `show you how.`?

Sorry I missed this the first time.

Comment (ryanseys) [inline README.md]: Ah, good example where my self-diagnosed ADHD kicked in and I got bored of writing that sentence.

Comment (stephenplusplus) [inline README.md]: Hahah, happens to me all the time.

**Comment by ryanseys** [comment]
NEW FEATURE: Complete sentences! Self-merging this.

**Comment by stephenplusplus** [comment]
:dancers:

**Pull request merged**